stock_analytic/tests/test_stock_picking.py

```diff
@@ -85,12 +106,12 @@ def _create_picking(
         return picking
 
     def _confirm_picking_no_error(self, picking):
-        picking.action_confirm()
+        picking.action_assign()
```

Comment:
@aheficent Why this change does not affect state ?

Reply:
I need to recheck because I don't remember well; but I think action_confirm does not exist in v12 and assigning will auto confirm the picking if stock available

Reply:
https://github.com/odoo/odoo/blob/12.0/addons/stock/models/stock_picking.py#L547

Reply:
I was wrong. Thanks!